pkg/minikube/cluster/cluster_test.go

```diff
@@ -50,3 +53,52 @@ func TestLocalBoot2DockerURL(t *testing.T) {
 		t.Fatalf("Expected URL : %s", localISOUrl)
 	}
 }
+
+func TestHostGetLogs(t *testing.T) {
+	api := tests.NewMockAPI()
```

Comment:
I don't really think this test serves any purpose here. It's a lot of code barely verifying anything. I suggest we remove this test.